src/components/Book/booking/booking.css

```diff
@@ -1,3 +1,20 @@
-#catButton {
-  font-size: 15px;
+#ulcategory {
+  max-height: 236px;
+  background-color: #c72121;
+  color: #fff;
+  overflow-y: auto;
+  width: fit-content;
+  border-radius: 5px;
+  padding: 10px;
+  margin: 0 auto;
+}
+
+#category {
```

Comment:
**You did a great job, however, some places need to be fixed.**
_Kindly follow Good and Bad CSS practices ([article for support](https://speckyboy.com/good-bad-css-practices))._ 

- [ ] Which is kindly use `classes` instead of `id` when you are selecting your element :)